### PR TITLE
update interop claims ISIKPatient + other Profiles PTData-378

### PR DESCRIPTION
**Pull request by f-peverali**
# Contributor Pull Request
Interop Claims are not up to date and must be reworked

## Description
Changed the Claims on Compatibilit (WIP
* ISIKPAtient
*  tbd. for ohters

## Motivation and Context
Claims on interop with other profiles  should be reliable
Answers: PTDATA-647

## How has this been tested?
No testing or validation, yet

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue) ?
- [x] Possible should be introduced as TC, because of information value ?

## Checklist:
- [x] I have already updated the documentation / narrative (intend) accordingly.

### CODE REVIEW

File: ImplementationGuide/markdown/Patient/Patient_Kompatibilitaet.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
 ### Kompatibilität
 
-Dieses Profil basiert auf dem [Profil KBV_PR_Base_Patient der KBV Basisprofile](https://simplifier.net/base1x0/kbv_pr_base_patient).
+Dieses Profil basierte ursprünglich auf dem [Profil KBV_PR_Base_Patient der KBV Basisprofile](https://simplifier.net/base1x0/kbv_pr_base_patient) in der Version 1.2.1.
```
Comment (f-peverali) [inline ImplementationGuide/markdown/Patient/Patient_Kompatibilitaet.md:3]:
```suggestion
```

File: ImplementationGuide/markdown/Patient/Patient_Kompatibilitaet.md [hunk 2]
```diff
@@ -1,16 +1,17 @@
 ### Kompatibilität
 
-Dieses Profil basiert auf dem [Profil KBV_PR_Base_Patient der KBV Basisprofile](https://simplifier.net/base1x0/kbv_pr_base_patient).
 
-Instanzen, die gegen ISiKPatient valide sind, sind auch (mit wenigen Einschränkungen) valide gegen:
+Für das Profil ISIKPatient wird eine Kompatibilität mit folgenden Profilen angestrebt - allerdings kann nicht sichergestellt werden, dass Instanzen, die gegen ISIKPatient valide sind, auch valide sind gegen:
 * [Profil KBV_PR_Base_Patient der KBV Basisprofile - v1.2.1](https://simplifier.net/packages/kbv.basis/1.2.1)
```
Comment (f-peverali) [inline ImplementationGuide/markdown/Patient/Patient_Kompatibilitaet.md:5]:
```suggestion
* [Profil KBV_PR_Base_Patient der KBV Basisprofile](https://fhir.kbv.de/StructureDefinition/KBV_PR_Base_Patient)
```

File: ImplementationGuide/markdown/Patient/Patient_Kompatibilitaet.md [hunk 3]
```diff
@@ -1,16 +1,17 @@
 ### Kompatibilität
 
-Dieses Profil basiert auf dem [Profil KBV_PR_Base_Patient der KBV Basisprofile](https://simplifier.net/base1x0/kbv_pr_base_patient).
 
-Instanzen, die gegen ISiKPatient valide sind, sind auch (mit wenigen Einschränkungen) valide gegen:
-* [Profil KBV_PR_Base_Patient der KBV Basisprofile - v1.2.1](https://simplifier.net/packages/kbv.basis/1.2.1)
+Für das Profil ISIKPatient wird eine Kompatibilität mit folgenden Profilen angestrebt - allerdings kann nicht sichergestellt werden, dass Instanzen, die gegen ISIKPatient valide sind, auch valide sind gegen:
+* [Profil KBV_PR_Base_Patient der KBV Basisprofile](https://fhir.kbv.de/StructureDefinition/KBV_PR_Base_Patient)
   * hier ist zu beachten, dass die KBV im Patienten-Profil für Address.country ein "extensible"-Binding auf die DEÜV-Codes festlegt; dies weicht von der ISiK Verwendung von ISO-Codes ab
 * [Profil Patient im International Patient Summary (IPS) - STU-1](https://hl7.org/fhir/uv/ips/StructureDefinition-Patient-uv-ips.html)
```
Comment (f-peverali) [inline ImplementationGuide/markdown/Patient/Patient_Kompatibilitaet.md:7]:
```suggestion
* [Profil Patient im International Patient Summary (IPS)](https://hl7.org/fhir/uv/ips/StructureDefinition-Patient-uv-ips.html)
```

File: ImplementationGuide/markdown/Patient/Patient_Kompatibilitaet.md [hunk 4]
```diff
@@ -1,16 +1,17 @@
 ### Kompatibilität
 
-Dieses Profil basiert auf dem [Profil KBV_PR_Base_Patient der KBV Basisprofile](https://simplifier.net/base1x0/kbv_pr_base_patient).
 
-Instanzen, die gegen ISiKPatient valide sind, sind auch (mit wenigen Einschränkungen) valide gegen:
-* [Profil KBV_PR_Base_Patient der KBV Basisprofile - v1.2.1](https://simplifier.net/packages/kbv.basis/1.2.1)
+Für das Profil ISIKPatient wird eine Kompatibilität mit folgenden Profilen angestrebt - allerdings kann nicht sichergestellt werden, dass Instanzen, die gegen ISIKPatient valide sind, auch valide sind gegen:
+* [Profil KBV_PR_Base_Patient der KBV Basisprofile](https://fhir.kbv.de/StructureDefinition/KBV_PR_Base_Patient)
   * hier ist zu beachten, dass die KBV im Patienten-Profil für Address.country ein "extensible"-Binding auf die DEÜV-Codes festlegt; dies weicht von der ISiK Verwendung von ISO-Codes ab
```
Comment (f-peverali) [inline ImplementationGuide/markdown/Patient/Patient_Kompatibilitaet.md:6]:
```suggestion
```

**Review by patrick-werner — APPROVED** [review]

File: ImplementationGuide/markdown/KontaktGesundheitseinrichtung/Kontakt_Kompatibilitaet.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 ### Kompatibilität
 
-Dieses Profil basiert auf dem Profil [KontaktGesundheitseinrichtung der Medizininformatik-Initiative](https://simplifier.net/medizininformatikinitiative-modulfall/kontaktgesundheitseinrichtung). 
+Für das Profil ISiKKontaktGesundheitseinrichtung wird eine Kompatibilität mit folgenden Profilen angestrebt; allerdings kann nicht sichergestellt werden, dass Instanzen, die gegen ISIKPatient valide sind, auch valide sind gegen:
```
Comment (patrick-werner) [inline ImplementationGuide/markdown/KontaktGesundheitseinrichtung/Kontakt_Kompatibilitaet.md:3]: Hier ist Kontakt statt ISiKPatient gemeint, oder?

File: ImplementationGuide/markdown/Prozedur/Prozedur_Kompatibilitaet.md [hunk 6]
```diff
@@ -1,10 +1,8 @@
 ### Kompatibilität
 
-Dieses Profil basiert auf dem [Profil "Prozedur" der Medizininformatik-Initative](https://simplifier.net/MedizininformatikInitiative-ModulProzeduren/~introduction). 
+Für das Profil ISIKPatient wird eine Kompatibilität mit folgenden Profilen angestrebt; allerdings kann nicht sichergestellt werden, dass Instanzen, die gegen ISIKPatient valide sind, auch valide sind gegen:
```
Comment (patrick-werner) [inline ImplementationGuide/markdown/Prozedur/Prozedur_Kompatibilitaet.md:3]: Prozedur statt Patient

File: ImplementationGuide/markdown/Versicherungsverhaeltnis/Versicherungsverhaeltnis_Kompatibilitaet.md [hunk 7]
```diff
@@ -3,24 +3,17 @@
 #### Profil ISiKVersicherungsverhaeltnisGesetzlich
 
 Das Profil ISiKVersicherungsverhaeltnisGesetzlich basiert auf dem Profil ["Coverage, deutsches GKV Profil" der deutschen Basisprofile](https://fhir.de/StructureDefinition/coverage-de-gkv).
```
Comment (patrick-werner) [inline ImplementationGuide/markdown/Versicherungsverhaeltnis/Versicherungsverhaeltnis_Kompatibilitaet.md:5]: es basiert auf:` http://fhir.de/StructureDefinition/coverage-de-basis` Coverage, deutsches Basisprofil.

File: ImplementationGuide/markdown/KontaktGesundheitseinrichtung/Kontakt_Kompatibilitaet.md (hunk 5 above)
Comment (f-peverali) [inline ImplementationGuide/markdown/KontaktGesundheitseinrichtung/Kontakt_Kompatibilitaet.md:3]:
```suggestion
Für das Profil ISiKKontaktGesundheitseinrichtung wird eine Kompatibilität mit folgenden Profilen angestrebt; allerdings kann nicht sichergestellt werden, dass Instanzen, die gegen ISiKKontaktGesundheitseinrichtung valide sind, auch valide sind gegen:
```

**Pull request merged**